content/docs/user-guide/troubleshooting.md

```diff
@@ -1,6 +1,7 @@
-# Troubleshooting
+# Troubleshooting Common DVC Issues
```

Comment:
let's not expand PR please 🙏 - it's not related in any way (is it needed at all?)

Reply:
Undone. Probably mistaken commit meant for my generic `jorge` branch... Sry